tools/ocean/divandfull.xml

```diff
@@ -2,7 +2,7 @@
     <description>Data-Interpolating Variational Analysis in n dimensions</description>
     <macros>
         <token name="@TOOL_VERSION@">0.1.0</token>
-        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@VERSION_SUFFIX@">0.0.1</token>
```

Comment:
its a simple integer, 0 ... 1... 2...3

Reply:
Oh I thought it follows the rule when you have minor fixes 0.0.+1, update 0.+1 or major changes +1